ui/app/templates/components/client-node-row.hbs

```diff
@@ -16,9 +16,9 @@
 <td data-test-client-datacenter>{{node.datacenter}}</td>
 <td data-test-client-volumes>{{if node.hostVolumes.length node.hostVolumes.length}}</td>
 <td data-test-client-allocations>
-  {{#if node.allocations.isPending}}
+  {{#if node.runningAllocations.isPending}}
```

Comment:
I mentioned this in the other PR too, but since `runningAllocations` returns an array instead of a PromiseProxy like the `allocations` `hasMany` relationship does, this will always be false.

This line should remain `node.allocations.isPending` to satisfy the loading state.